guides/file-uploads.md

```diff
@@ -51,3 +51,43 @@ we were merely putting them in the context as in other implementations.
 * Apollo: [apollo-absinthe-upload-link](https://www.npmjs.com/package/apollo-absinthe-upload-link)
 * Apollo (v1): [apollo-absinthe-upload-client](https://www.npmjs.com/package/apollo-absinthe-upload-client) (Note: does not support Relay Native as of v1.0.1)
 * Relay: _(None known. Please submit a pull request updating this information.)_
+
+## Testing
+
+Use `%Plug.Upload{}` struct to stub your uploaded file.
+
+```Elixir
+defmodule API.UploadFileMutationTest do
+  use API.ConnCase
```

Comment:
I'm not clear what this is... Ideally an example test will use generic testing mechanisms so that anyone could re-use it...